docs_user/modules/proc_stopping-infrastructure-management-and-compute-services.adoc

```diff
@@ -1,21 +1,19 @@
 [id="stopping-infrastructure-management-and-compute-services_{context}"]
 
-//:context: stopping-infrastructure-management
-//kgilliga: This module might be converted to an assembly, or a procedure as a standalone chapter.
-
-= Stopping infrastructure management
+= Stopping infrastructure management and Compute services
 
 The source cloud's control plane can be decomissioned,
 which is taking down only cloud controllers, database and messaging nodes.
-Nodes that must remain functional are those running the compute, storage,
+Nodes that must remain functional are those running the Compute, storage,
 or networker roles (in terms of composable roles covered by Tripleo Heat
 Templates).
 
-== Variables
+.Variables
 
-Define the shell variables used in the steps below.
-Define the map of compute node name, IP pairs.
-The values are just illustrative and refer to a single node standalone director deployment, use values that are correct for your environment:
+Define the following shell variables.
+Define the map of Compute node name, IP pairs.
+The values are just illustrative and refer to a single node standalone {OpenStackPreviousInstaller} deployment, use values that are correct for your environment: 
+//kgilliga: Is this correct? Standalone director deployment for downstream, standalone tripleo for upstream?
```

Comment:
this guide does only cover standalone tripleo, which could also be an all-in-one overlcoud managed by OSP director